modules/ROOT/pages/visualizer-app-network.adoc

```diff
@@ -19,97 +17,43 @@ When viewing or modifying a graph, Anypoint Visualizer provides the following fu
 
 * Zoom control: Use the zoom in, zoom out, and zoom fit buttons on the bottom right corner of the canvas to control the zoom. You can also use scroll to zoom in and out.
 * Pan the visualization: Click and drag on the canvas to pan the visualization.
+* Search: Search for the name of a service on the canvas and the nodes that contain your search will be highlighted.
+
+== Select your Use Case
+
+Anypoint Visualizer has three use cases, or _visualizations_: Architecture, Troubleshooting, and Policies. By identifying your use case, you can select the visualization that suits your needs.
+
+The xref:architecture-visualization.adoc[_Architecture Visualization_] shows the topology of your apps and APIs. This visualization is the most useful for understanding your system architecture. You can view and modify the names of your services, give them specific tags, and assign them to layers.
+
+The xref:troubleshooting-visualization.adoc[_Troubleshooting Visualization_] enables you to view the health of the system and evaluate performance-based metrics. This visualization helps you identify issues or unusual behavior across your apps and APIs.
+
+The xref:policy-visualization.adoc[_Policy Visualization_] enables you to see the policy landscape of your application network at a glance. You can view which components are compliant with policies you select, and you can determine where lapses in security or policy application may have occurred.
 
 == View an Application Network in an Organization
 
 . From Management Center, select *Visualizer*.
 +
 If Visualizer is not available, contact your system administrator to ensure that you have the correct permissions to access the application.
 
+. In the drop-down in the canvas, select the visualization you want to use.
```

Comment:
Do not use "drop-down" alone for indicating a list or menu. In fact, if you can avoid it, omit "drop-down" altogether than use only "list" or "menu." Our audience should know what that downward-facing arrow indicates....

Reply:
```suggestion
. In the list in the canvas, select the visualization you want to use.
```